pkg/skaffold/initializer/testdata/init/getting-started-kustomize/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+  - deployment.yaml
+patches:
```

Comment:
Nit: File lists are deprecated for patches.

Reply:
and we still have them in all of our examples :)